docs/_includes/Types.md

```diff
@@ -2957,16 +2952,17 @@ known constants at compile-time. They need not be numeric literals; combinations
 of basic operations and symbolic constants are also allowed as described
 in [Section: Compile-Time Constants](#sec-compile-time-constants).
 
-## Numeric conversion operations
+## Conversion operations {#sec-conversion}
 
-For every numeric type `N`, there is a conversion function with the
-name `as N`.  It is a partial identity function.  It is defined when the
-given value, which can be of any numeric type, is a member of the type
+For every type `N`, there is a conversion operation with the
+name `as N`, described more fully in [a later section](#sec-as-expression).
```

Comment:
This is not yet supported for all types. Is this sentence perhaps forward-looking in the same way as the description of `is` (see L2991)?

Reply:
Yes -- and I mention just a bit later that this is only implemented for numeric base types so far.